confgenerator/config.go

```diff
@@ -273,11 +279,59 @@ func newValidator() *validator.Validate {
 		}
 		return t%tfactor == 0
 	})
+	// winlogchannels wraps validateWinlogchannels when receiver_version > 1
```

Comment:
Why can't we match exactly 2 as the expected winlog version?

Reply:
Only in the comment that mentions >1, why can't we just say == 2, i saw the validator uses "oneof 1 2"